src/main/java/com/quantumlytangled/gravekeeper/GraveKeeperConfig.java

```diff
@@ -297,5 +301,11 @@ public static void onFMLpostInitialization() {
         SOULBOUND_ENCHANTMENTS.add(enchantment);
       }
     }
+
+    SOULBOUND_TAGS = new HashMap<>();
+    for (final String tagValue : SOULBOUND_TAG_VALUE) {
+      final String[] tagSplit = tagValue.split(":");
+      SOULBOUND_TAGS.put(tagSplit[0], Boolean.parseBoolean(tagSplit[1]));
```

Comment:
There's no warranty that tagSplit[1] exists nor can be parsed as Boolean. Consider adding a try/catch with an appropriate error message.